src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -31,11 +31,16 @@ public Drivetrain() {
    *                        drivetrain.
    * @param rotationSpeed   The rotation speed to apply to the drivetrain.
    */
-  public void setDrivetrainSpeed(double forwardVelocity, double rotationSpeed) {
+  public void setDrivetrainSpeed(double forwardVelocity, double rotationSpeed, boolean leftBumperDown) {
```

Comment:
We want out parameter to say what they do in the function rather than what will be passed to them (we may change bumpers for example)
```suggestion
  public void setDrivetrainSpeed(double forwardVelocity, double rotationSpeed, boolean slowMode) {
```